tekton/pipeline_run.go

```diff
@@ -70,16 +71,16 @@ func (r *ReleasePipelineRun) AsPipelineRun() *tektonv1beta1.PipelineRun {
 	return &r.PipelineRun
 }
 
-// WithEnterpriseContractConfigMap adds a param providing the verify ec task bundle to the release PipelineRun.
+// WithEnterpriseContractConfigMap adds a param providing the verify ec task git resolver information to the release PipelineRun.
 func (r *ReleasePipelineRun) WithEnterpriseContractConfigMap(configMap *corev1.ConfigMap) *ReleasePipelineRun {
```

Comment:
Could we use this PR to give a better name to the parameter? Something like `ecConfig` or similar?

Reply:
You think changing the function name to replace `EnterpriseContract` with `ec` makes it better? I don't

Reply:
I think David meant the argument of this function: `configMap` -> `ecConfig`

Reply:
got it, thanks for clarifying guys. Updated